app-crypt/rnp/rnp-0.15.0.ebuild

```
@@ -0,0 +1,48 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+DESCRIPTION="high performance C++ OpenPGP library, fully compliant to RFC 4880"
+HOMEPAGE="https://www.rnpgp.org/"
+SRC_URI="https://codeload.github.com/rnpgp/rnp/tar.gz/refs/tags/v${PV} -> ${P}.tar.gz"
```

No functional change but it matches just about every other ebuild

```suggestion
SRC_URI="https://github.com/rnpgp/rnp/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
```